guides/creating-a-custom-stepper-using-the-cdk-stepper.md

```diff
@@ -0,0 +1,187 @@
+# Creating a custom stepper using the CDK stepper
+
+Using the [CDK stepper](https://material.angular.io/cdk/stepper/overview) it is possible to build a custom stepper which you can completely style yourself without any specific Material Design styling.
```

Comment:
Try to avoid [passive voice](https://en.wikipedia.org/wiki/English_passive_voice), e.g. instead of writing
"Using the CDK stepper it is possible...", you could write "The CDK stepper allows..."

Reply:
Done